draft-ietf-wish-whip.md

```diff
@@ -155,9 +155,9 @@ Location: https://whip.example.org/resource/id
 <SDP answer>
 ~~~~~
 
-A WHIP client sending a PATCH request for performing trickle ICE MUST contain an If-Match header with the latest known entity-tag as per {{!RFC7232}} section 3.1. When the PATCH request is received by the WHIP resource, it MUST compare the entity-tag value requested with the current entinty-tag of the resourceas per {{!RFC7232}} section 3.1 and return a 412 Precondition Failed response if they do not match.
+A WHIP client sending a PATCH request for performing trickle ICE MUST contain an If-Match header with the latest known entity-tag as per {{!RFC7232}} section 3.1. When the PATCH request is received by the WHIP resource, it MUST compare the entity-tag value requested with the current entinty-tag of the resourceas per {{!RFC7232}} section 3.1 and return a 412 Precondition Failed response if they do not match. Entity-tag validation MUST only be used for HTTP request requiring to match a known ICE session  and MUST NOT be used otherwise, for example in the HTTP DELETE request to terminate the session.
```

Comment:
I'd change the MUST NOT to a SHOULD NOT, as the MUST NOT sounds a bit too constraining perhaps?

Reply:
ok